geoglows/data.py

```
@@ -228,3 +301,36 @@ def metadata_tables(columns: list = None, metadata_table_path: str = None) -> pd
+    if saber_table_path:
+        return pd.read_parquet(saber_table_path, columns=columns)
+    saber_table_path = get_saber_assign_table_path()
+    if os.path.exists(saber_table_path):
+        return pd.read_parquet(saber_table_path, columns=columns)
+
+    warn = f"""
+    Local copy of SABER assign table not found.
+    A copy of the table has been cached at {saber_table_path} which you can move as desired.
+    You should set the environment variable PYGEOGLOWS_SABER_ASSIGN_TABLE_PATH or provide the saber_table_path argument.
+    """
+    warnings.warn(warn)
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)